jwt/validation.go

```diff
@@ -47,6 +47,14 @@ func (e Expected) WithTime(t time.Time) Expected {
 
 // Validate checks claims in a token against expected values.
 // A default leeway value of one minute is used to compare time values.
+//
+// The leeway will cause the token to be deemed valid until one minute
```

Comment:
Maybe write "default leeway" here to indicate that that's just the default value, not a fixed value?